mobile_app/path_constants.py

```diff
@@ -0,0 +1,9 @@
+"""
+Path constants used by the mobile build scripts
+"""
+CONFIG_FILE = "CONFIGURATION"
+
+CODE_CHECKOUT = "code.git"
```

Comment:
Suggestion to rename these to CHECKOUT_DIRECTORY_NAME so it's clear it's the name of the local directory to where they are checked out.